bin/taxref_reformat_sbdi-gtdb.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Formatting script for sbdi-gtdb files.
+
+# We get the files with numbers as names. Loop over them and find out which 
+# looks like assignTaxonomy and addSpecies respectively, gunzip to proper
+# names.
+
+for f in *; do
+  if [ $(file -L $f | grep -c 'gzip') ]; then
+    if [ $(gunzip -c $f | head -n 1 | grep -c '>.*;.*;.*;.*;.*') -eq 1 ]; then
+      gunzip -c $f > assignTaxonomy.fna
+    elif [ $(gunzip -c $f | head -n 1 | grep -c '>.* .*') -eq 1 ]; then
+      gunzip -c $f > addSpecies.fna
+    fi
+  fi
```

Comment:
```suggestion
for f in *; do
    if [ $(file -L $f | grep -c 'gzip') ]; then
        if [ $(gunzip -c $f | head -n 1 | grep -c '>.*;.*;.*;.*;.*') -eq 1 ]; then
            gunzip -c $f > assignTaxonomy.fna
        elif [ $(gunzip -c $f | head -n 1 | grep -c '>.* .*') -eq 1 ]; then
            gunzip -c $f > addSpecies.fna
        fi
    fi
```

Reply:
I think that should solve it, a little hard to do that in the rudimentary github editor.

Reply:
Sorry that you had to do that. I didn't notice until I had committed an update myself :-S

Reply:
No problem!